modules/base/rendering/renderablemodel.cpp

```diff
@@ -169,14 +171,14 @@ namespace {
     constexpr openspace::properties::Property::PropertyInfo EnableDepthTestInfo = {
         "EnableDepthTest",
         "Enable Depth Test",
-        "Enable Depth Testing for the Model.",
+        "If true, depth testing is enabled for the model.",
```

Comment:
Maybe adding a sentence or two in here what it means for Depth Testing to be enabled or disabled?

Reply:
@Roxeena could you maybe help out with this?  Looking at the special handling in the shader and not exactly sure how to describe it

Reply:
So, should I add a sentence or two based on @Roxeena's clarification, or should we remove it? @alexanderbock @Roxeena

Reply:
Yea, adding a sentence or two describing it would be great